### DISCUSSION TITLE
Fix using/explain using Copilot taking precedence over import actions

### DISCUSSION THREAD

**Discussion by ytanay** [community/community#68025] [Copilot Conversations]
### Select Topic Area

Bug

### Body

It now takes considerably more keystrokes to add a missing import (e.g. in Rust) due to the "Fix with Copilot" and "Explain using Copilot" actions.

![image](https://github.com/community/community/assets/3638334/bcd92e5e-4c9a-4898-bde6-a2f091aef633)

Is there a way to reorder these actions?

**Comment by joshka** [comment]
+1 This should be added after the existing fixes, not before.
